lib/internal/test_runner/reporter/utils.js

```diff
@@ -84,10 +84,15 @@ function formatTestReport(type, data, prefix = '', indent = '', hasChildren = fa
   return `${prefix}${indent}${color}${symbol}${title}${colors.white}${err}`;
 }
 
+function reRunTestFileMessage(file) {
+  return `\n[${new Date().toISOString()}] Restarting ${file}\n`
```

Comment:
About this: I might be wrong, but in case of an update to a dependency, more than one file could be restarted at the same time.  
For this reason, I'd avoid referencing a specific file!